core/modules/file/file.module

```diff
@@ -1316,6 +1316,9 @@ function file_managed_file_process($element, &$form_state, $form) {
   );
 
   // Add a button that will be used to select existing files if enabled.
+  // Attaching the assets from filter module makes sure, the browser works
+  // even if no field on that page has a filter set.
```

Comment:
The comma feels out of place here. Perhaps it was meant to go after the word "works"(?).

While at it, can we please be more detailed? How about this?:

```suggestion
  // Attaching the assets from the Filter module makes sure that the image
  // browser works, even if no field on that form has a filter set. This can
  // happen on forms that have only image field(s), without any WYSIWYG-enabled
  // text field.
```

Reply:
WYSIWYG-enabled is misleading here. The problem has nothing to do with the editor, but with a filter (text format) applied or not applied.